echo-extended/server/echo-extended.py

```diff
@@ -55,8 +76,8 @@ def advance():
     if transform not in transformations:
         error_message = "Cannot do the '" + transform + "' transformation"
         error_serialize = serialize(error_message)
-        requests.post(dispatcher_url + "/notice", json={"payload": error_message})
-        requests.post(dispatcher_url + "/finish", json={"status": "accept"})
+        requests.post(dispatcher_url + "/report", json={"payload": error_message})
```

Comment:
I guess you just forgot to update this line to use `error_serialized` instead of `error_message` :wink: